docs/src/main/sphinx/language/types.md

```diff
@@ -155,6 +155,9 @@ before any Unicode character usage with 4 digits. In the examples above
 with 6 digits require usage of the plus symbol before the code. For example,
 you need to use `\+01F600` for a grinning face emoji.
 
+Single quotes in string literals can be escaped by using another single quote: 
```

Comment:
While true as it stands.. I am not sure this is the correct spot in the docs .. it also applies to char at least ..

We also might want to mention using double quotes for identifiers somewhere .. unless we do already

Reply:
Identifiers were covered in https://github.com/trinodb/trino/pull/18181
When working on that we decided not to have a separate page for generic sql syntax and put all literals descriptions here in data types. I agree we might mention this in chars too, but that should be it.

Reply:
Oh well, maybe also in the query table function docs. Some people asked about it in the Trino slack, how to escape quotes around literals in query "parameters"